verilog/tools/kythe/verilog_kythe_extractor.cc

```diff
@@ -50,10 +45,28 @@ int ExtractOneFile(absl::string_view content, absl::string_view filename) {
 
     std::cout << facts_tree << '\n';
   }
+  LOG(INFO) << '\n' << facts_tree;
+
+  // check for printkythefacts flag, and print the facts if on
+  if (absl::GetFlag(FLAGS_printkythefacts)) {
+    std::cout << std::endl
+              << (!parse_ok ? " (incomplete due to syntax errors): " : "")
+              << std::endl;
+
+    std::cout << verilog::kythe::KytheFactsPrinter(facts_tree) << '\n';
+  }
 
   return exit_status;
 }
 
+// verilog_kythe_extractor is a simple command-line utility to extract kythe
+// indexing facts from the given file.
+//
+// Expected Input: verilog file.
+// Expected output: Produces Indexing Facts for kythe.
+//
+// Example usage:
+// verilog_kythe_extractor files...
```

Comment:
Can you move this into the `StrCat` args for `usage`?  This will then appear when you run the binary with `--help`.